todo/views.py

```diff
@@ -1,3 +1,19 @@
 from django.shortcuts import render
+from todo.models import Todo
+from django.contrib.auth.models import User
+from todo.forms import TodoSettingForm
 
-# Create your views here.
+def add_todo(request):
+    """
+    formの情報をtodoテーブルに格納．
+    ログインしているuser_idと紐付ける．
+    """
+    form = TodoSettingForm(request.POST or None)
+    # fieldの値が正しいかを検証
+    if form.is_valid():
```

Comment:
SHOULD: この実装だと、初回アクセス時にも `is_valid()` がFalseになって `form.errors` が設定されてしまいます。以下の様にGETとPOSTで分けて実装することをお勧めします。

```suggestion
    if request.method == "POST" and form.is_valid():
```

参考: https://docs.djangoproject.com/ja/3.0/topics/forms/#the-view